lib/logger.js

```diff
@@ -234,6 +244,10 @@ function logToConsole(type, sender, msgShort, _extras) {
   if (!checkType(type)) return;
   if (!logThis(type)) return;
 
+  if (_extras instanceof Error) {
```

Comment:
I don't think it is a good approach to handle errors and log them in Sentry through some shared package.

Reply:
what problem do you see?
the current thing is that we write the same code over and over. we could hide it in a function within the same repo but then we copy past the same function to x repos and need to maintain them.

from a clean arch perspective this is polluting our code base it has no business value but as a developer you need to know about it.

Reply:
Configuration: Each project may have different requirements for logging, such as different log levels or different destinations for logs. If the Sentry logger is defined in a shared package, it may not be possible to configure it differently for each project.

Maintenance: If the Sentry logger is defined in a shared package, any changes to the logger will affect all projects that use the package. This can make it difficult to maintain the logger and ensure that it works correctly for each project.

Dependencies: The Sentry logger may have dependencies on other packages or libraries that are not compatible with all projects. If the logger is defined in a shared package, it may be difficult to manage these dependencies and ensure that they work correctly for each project.

Reply:
configuration: I guess answered in the other comment
maintenance: 4 changes in 3 years... I'm willing to take the risk^^
dependencies: versioning